ScoutSuite/cli_parser.py

```diff
@@ -1,264 +1,339 @@
                                       help='Run Scout Suite with Managed Service Identity')
-        azure_auth_modes.add_argument('--azure-service-principal',
+
+        azure_auth_modes.add_argument('--service-principal',
                                       action='store_true',
                                       help='Run Scout Suite with an Azure Service Principal')
-        azure_auth_modes.add_argument('--azure-file-auth',
-                                      action='store',
-                                      type=argparse.FileType('r'),
-                                      metavar="FILE",
-                                      help='Run Scout Suite with the specified credential file')
-        azure_auth_modes.add_argument('--azure-user-credentials',
-                                      action='store_true',
-                                      help='Run Scout Suite with user credentials')
-
         azure_auth_params.add_argument('--tenant',
                                        action='store',
                                        dest='tenant_id',
-                                       help='Tenant ID of the Azure service principal')
+                                       help='Tenant ID of the service principal')
         azure_auth_params.add_argument('--subscription',
                                        action='store',
                                        dest='subscription_id',
-                                       help='Subscription ID of the Azure service principal')
+                                       help='Subscription ID of the service principal')
         azure_auth_params.add_argument('--client-id',
                                        action='store',
                                        dest='client_id',
-                                       help='Client ID of the Azure service principal')
+                                       help='Client ID of the service principal')
         azure_auth_params.add_argument('--client-secret',
                                        action='store',
                                        dest='client_secret',
-                                       help='Client secret of the Azure service principal')
+                                       help='Client of the service principal')
+
+        azure_auth_modes.add_argument('--file-auth',
+                                      action='store',
+                                      type=argparse.FileType('r'),
+                                      dest='file_auth',
+                                      metavar="FILE",
+                                      help='Run Scout Suite with the specified credential file')
 
-        azure_auth_params.add_argument('--azure-username',
+        azure_auth_modes.add_argument('--user-account',
+                                      action='store_true',
+                                      help='Run Scout Suite with user credentials')
```

Comment:
Shouldn't this be Azure Account?

Reply:
I tried to keep the name consistant with GCP --user-account. Also, since the swtiches are independant, we don't need to precise which provider as they are already provider-specific

Reply:
My bad, I copy pasted this comment one too many times haha